packages/communication-react/CHANGELOG.beta.md

```diff
@@ -4,123 +4,71 @@ This log was last generated on Mon, 28 Aug 2023 22:17:59 GMT and should not be m
 
 <!-- Start content -->
 
-## [1.8.0-beta.1](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.8.0-beta.1)
+## [1.8.0-beta.1](https://github.com/azure/communication-ui-library/tree/1.8.0-beta.1)
+
+
 
 Mon, 28 Aug 2023 22:17:59 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.7.0...@azure/communication-react_v1.8.0-beta.1)
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.7.0...1.8.0-beta.1)
+
+### Mention - Public Preview
+The Web UI Calling composite now supports Raise Hand capabilities. This enables users to raise and lower hand in Teams interop meetings.
```

Comment:
Why Raise Hand under the Mention heading? Also is it possible to change Raise Hand capabilities -> Raise Hand. The Capabilities feature can be confused here and the capabilities feature for raise hand is something we are not doing yet.

Reply:
nit: Calling composite -> Call composite

Reply:
I'll change the wording around. Good find, this was a copy pasta error.